src/autocomplete/EmojiProvider.tsx

```diff
@@ -34,7 +34,7 @@ const LIMIT = 20;
 
 // Match for ascii-style ";-)" emoticons or ":wink:" shortcodes provided by emojibase
 // anchored to only match from the start of parts otherwise it'll show emoji suggestions whilst typing matrix IDs
-const EMOJI_REGEX = new RegExp('(' + EMOTICON_REGEX.source + '|(?:^|\\s):[+-\\w]*:?)$', 'g');
+const EMOJI_REGEX = new RegExp('(' + EMOTICON_REGEX.source + '|(?:^|\\s|(?<=^\\+)):[+-\\w]*:?)$', 'g');
```

Comment:
Please add a comment above this regex similar to `REGEX_EMOTICON_WHITESPACE`, so it's easier to follow what's happening here in the future.